pylint/checkers/base.py

```diff
@@ -2210,17 +2190,21 @@ def visit_functiondef(self, node: nodes.FunctionDef) -> None:
                         overridden = True
                         break
                 self._check_docstring(
-                    ftype, node, report_missing=not overridden, confidence=confidence
+                    ftype, node, report_missing=not overridden, confidence=confidence  # type: ignore
```

Comment:
Although `VS Code` correctly infers that `ftype` is a string literal because of line 2170, `mypy` does not.